puterbot/models.py

```diff
@@ -222,3 +222,11 @@ class WindowEvent(Base):
     top = sa.Column(sa.Integer)
     width = sa.Column(sa.Integer)
     height = sa.Column(sa.Integer)
+
```

Comment:
Please add two newlines between classes

Reply:
Why is that?

Reply:
I see the file is like that already, done in latest commit.

Reply:
Thanks @apgorton ! See https://stackoverflow.com/a/47448739/95989 . From [PEP8](https://peps.python.org/pep-0008/#imports):

> Surround top-level function and class definitions with two blank lines.